service/src/model/durations.rs

```diff
@@ -601,10 +377,13 @@ where
 
 impl<D: sea_orm::TryGetable> sea_orm::TryGetable for PossiblyInfinite<D> {
     fn try_get_by<I: sea_orm::ColIdx>(
-        res: &sea_orm::prelude::QueryResult,
+        res: &sea_orm::QueryResult,
         index: I,
     ) -> Result<Self, TryGetError> {
-        let value: String = res.try_get_by(index).map_err(TryGetError::DbErr)?;
+        let value = res
+            .try_get_by(index)
+            .map_err(TryGetError::DbErr)
+            .and_then(|opt: Option<String>| opt.ok_or(TryGetError::Null(format!("{index:?}"))))?;
```

Comment:
This was causing SeaORM to fail when reading optional durations from the database…